intents/intent.ridl

```diff
@@ -0,0 +1,183 @@
+webrpc = v1
+
+name = sequence-waas-intents
+version = v0.1.0
+
+struct Intent
+  - version: string
+  - name: string
+  - expiresAt: uint64
+  - issuedAt: uint64
+  - data: any
+  - signatures: []Signature
+    + go.tag.json = ,omitempty
+
+struct Signature
+  - sessionId: string
+     + go.field.name = SessionID
+  - signature: string
+
+# no way to generate string enums
+# enum IntentName: string
+#  - openSession
+#  - closeSession
+#  - validateSession
+#  - finishValidateSession
+#  - listSessions
+#  - getSession
+#  - signMessage
+#  - sendTransaction
+
+struct IntentDataOpenSession
+  - sessionId: string
+    + go.field.name = SessionID
+  - email?: string
+    + go.tag.json = ,omitempty
```

Comment:
`,omitemtpy` ...? good?

Reply:
Yep, this is good. This comma is required there. I know it looks off but it needs to be like this to avoid issues.